src/pages/AccountRequest/accountUtil.tsx

```diff
@@ -0,0 +1,55 @@
+import { IFormOption } from "components/Form/interfaces";
+
+export interface IUserFormValues {
+  username: string;
+  email: string;
+  full_name: string;
+  role_id: number;
+  introduction: string;
+  institution_id: number;
+}
+
+export enum EmailPreference {
+  EMAIL_ON_REVIEW = "email_on_review",
+  EMAIL_ON_SUBMISSION = "email_on_submission",
+  EMAIL_ON_META_REVIEW = "email_on_review_of_review",
+}
+
+export interface IRole {
+  id: number;
+  name: string;
+  parent_id: number;
+}
+
+export interface IInstitution {
+  id: number;
+  name: string;
+}
+
+export const emailOptions: IFormOption[] = [
```

Comment:
It seems like this code isn't even necessary. It would be best to only include the code that is required.